### ISSUE TITLE
Can't use SMTP due to deprecated parameter in SSL.pm

### ISSUE THREAD

**Issue by altinukshini** [mysociety/fixmystreet#1639]
Whenever I try to run send reports or try to create a new account under /auth I get this in logs:

*******************************************************************
 Using the default of SSL_verify_mode of SSL_VERIFY_NONE for client
 is deprecated! Please set SSL_verify_mode to SSL_VERIFY_PEER
 together with SSL_ca_file|SSL_ca_path for verification.
 If you really don't want to verify the certificate and keep the
 connection open to Man-In-The-Middle attacks please set
 SSL_verify_mode explicitly to SSL_VERIFY_NONE in your application.
*******************************************************************
  at /var/www/platform/fixmystreet/web/../local/lib/perl5/Email/Send/SMTP.pm line 59.

Apache log:

[Fri Feb 10 15:08:36.932178 2017] [fcgid:warn] [pid 4249:tid 140227360274176] [client 127.0.0.1:48210] mod_fcgid: stderr: [error] Caught exception in FixMyStreet::App::Controller::Auth->email_sign_in "invalid SSL_version specified at /var/www/platform/fixmystreet/web/../local/lib/perl5/IO/Socket/SSL.pm line 389.", referer: http://mywebsite.org/auth?r=my

**Comment by dracos** [comment]
Sorry to hear that; could you confirm what platform/OS you're running this on?

**Comment by altinukshini** [comment]
Ubuntu 16.04.1 LTS

**Comment by pauliuszaleckas** [comment]
I get the same error when using external SMTP with SSL.
I am using AMI 2017-02-03.

**Comment by pauliuszaleckas** [comment]
And I get the similar error with TLS also:
```
*******************************************************************
 Using the default of SSL_verify_mode of SSL_VERIFY_NONE for client
 is deprecated! Please set SSL_verify_mode to SSL_VERIFY_PEER
 together with SSL_ca_file|SSL_ca_path for verification.
 If you really don't want to verify the certificate and keep the
 connection open to Man-In-The-Middle attacks please set
 SSL_verify_mode explicitly to SSL_VERIFY_NONE in your application.
*******************************************************************
  at /var/www/fixmystreet/fixmystreet/bin/../local/lib/perl5/Net/SMTP/TLS.pm line 181.
invalid SSL_version specified at /var/www/fixmystreet/fixmystreet/bin/../local/lib/perl5/IO/Socket/SSL.pm line 389.
```

**Comment by pauliuszaleckas** [comment]
I did some workarounds, but e-mail sending still fails. Overall it looks like we should switch from Email::Send (deprecated) to Email::Sender. Net::SNMP, IO::Socket::SSL and maybe other packages will need update. Unfortunately I am very new on Perl, so I can't do this quick myself...

**Comment by dracos** [comment]
Hopefully I will have some time to look at this tomorrow. In the mean time, you should be able to set up a mail server running on localhost that sends mail onwards, and then send to that from FixMyStreet.

**Comment by dracos** [comment]
Hi, hopeful fix for this at #1673, we should be able to review/merge next week.
